.gitlab-ci/cv32.yml

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This file will not be necessary as the original version of it (`.gitlab-ci.yml`) is no longer in use.

Reply:
would you prefer I remove it or leave it as an entry point to motivate someone to help set up a cv32 regression?

Reply:
Originally I was thinking that it should be removed.  Your idea to leave it as an entry point for future cv32 regressions on from GitLab is a good one.  Perhaps add a comment to indicate that it is currently not used.  Thanks!